Feature/113

JaimeBorregoConde opened the pull request:
Test importación de censo

JaimeBorregoConde committed:
feat: Changes in import.html
Immport.html has been changed in order to include all text data for translation

JaimeBorregoConde committed:
Merge branch 'feature/027' of https://github.com/egc-decide-trebujena/decide into feature/086

JaimeBorregoConde committed:
fix: Improvement in error messages views
Import excel html has been modified to control message exception changes in import excel views

JaimeBorregoConde committed:
test: Positive and negative test for import census added
Selenium test has been created for testing the import of .xlsx census files

JaimeBorregoConde committed:
refactor: Refactor test suite
Function create_excel_file has been added to simplify import_excel_test.

JaimeBorregoConde committed:
Fix: Added line for fiz github actions
Added lien in .git/worflow/django.yml to fix it

JaimeBorregoConde committed:
Fix: Added requirement for actions
Last commit does not work well because requirement.txt was needed for actions

JaimeBorregoConde committed:
Fix: modified requirement for actions
Last commit does not work well because requirement.txt was needed for actions

JaimeBorregoConde committed:
Fix: Change in import excel test
 There was a error in one positive test. We where testing the same positive case two times.

gonmarfer2 reviewed (dismissed):
Todo oquey

JaimeBorregoConde committed:
Merge branch 'module/censo' into feature/113

gonmarfer2 reviewed (dismissed):
Oquey, una vez corrijas los conflictos, puedes fusionar la rama.

JaimeBorregoConde committed:
Merge branch 'module/censo' into feature/113

JaimeBorregoConde committed:
Fix: Fixed error with test while merging
Automatic github merge broke test. csv.html does not exist. Export test were deleted.

JaimeBorregoConde committed:
Merge branch 'module/censo' into feature/113

JaimeBorregoConde committed:
Fix: Migrations folder does not appear
Added migration folder for makemigrations

gonmarfer2 reviewed (approved):
Now, everything works
